packages/core/src/store.ts

```diff
@@ -65,41 +54,33 @@ export function createStore<
   const getState = () => currentState;
 
   const addUpdate = (
-    statePart: Partial<TState>,
+    updateAction: UpdateStateAction<TState>,
   ): void => {
     nextState = {
       ...nextState,
-      ...statePart,
+      ...updateAction(nextState),
     };
   };
 
-  const commitUpdates = () => {
+  const commitPropsUpdates = () => {
     currentState = nextState;
     baseCommitUpdate(validator(currentState));
```

Comment:
**Clarification, possible code changes**
Should we really call baseCommitUpdate?
I suppose it's called then we get new props and should sync it with a store. Isn't it? So if yes why do we call baseCommitUpdate what related to changes store by action and implement controlled mode?

Reply:
We discussed it in a live conversation.
And we decided to discuss this store updating mechanism in the planned meeting about the controlled mode.